docs/pages/management/admin/labels.mdx

```diff
@@ -1,405 +1,323 @@
-
-When configuring a command to generate dynamic labels, the executable named
-within your command must be the path to a valid file. The executable permission
-bit must be set and shell scripts must have a
-[shebang line](https://en.wikipedia.org/wiki/Shebang_\(Unix\)).
-
-</Notice>
-
-Once you have configured your labels, start Teleport with the invite token you
-created earlier:
-
-```code
-$ sudo teleport start --token=${INVITE_TOKEN?}
-```
-
-Verify that you have added the labels by running the following on your local
-machine. Your Teleport user must be authorized to access the Node.
-
-```code
-$ tsh ls
-Node Name         Address    Labels
------------------ ---------- ---------------
-ip-192-0-2-0      ⟵ Tunnel arch=x86_64,hostname=ip-172-30-156-233
-```
-
-<Details title="Problems re-joining the Node?">
-
-Try creating another invite token using the `tctl tokens add` command from earlier
-and starting your Node again.
-
-If this doesn't work, delete the directory your Node uses to maintain its state,
-`/var/lib/teleport`, and try again.
-
-</Details>
-
-## Next steps: Using labels
-
-Once you have labeled your resources, you can refer to your labels when running
-`tsh` and `tctl` commands to filter the resources that the commands will query.
-For more information, see
-[Resource filtering](../../reference/predicate-language.mdx).
+Teleport configuration file, then restarting the Teleport service on your server.
+
+To add a command to generate a dynamic label:
+
+1. Stop the Teleport service running on your server.
+
+1. Open the Teleport configuration file—by default, `/etc/teleport.yaml`—in a text editor.
+
+1. Locate the `commands` configuration under the `ssh_service` section.
+
+1. Add a `command` array that runs the `uname` command with the `-p` argument to return the architecture of the 
+host server every one hour. 
+   
+   For example, add the `name`, `command`, and `period` fields as follows:
+
+   ```yaml
+   ...
+   ssh_service:
+     enabled: "yes"
+     labels:
+       teleport.internal/resource-id: 1f2cdcc5-cde3-41fa-b390-bc872087821a
+       environment: dev
+     commands:
+     - name: hostname
+       command: [hostname]
+       period: 1m0s
+     - name: arch
+       command: [uname, -p]
+       period: 1h0m0s
+   ```
+   
+   In the `command` setting, the first element is a valid executable. 
+   Each subsequent element is an argument.
+   The following syntax is valid:
+
+   ```yaml
+   command: ["/bin/uname", "-m"]
+   ```
+   
+   The following syntax is not valid:
+
+   ```yaml
+   command: ["/bin/uname -m"]
+   ```
+   
+   For more complex commands, you can use single (') and double (") 
+   quotation marks interchangeably to create nested expressions. 
+   For example:
+
+   ```yaml
+   command: ["/bin/sh", "-c", "uname -a | egrep -o '[0-9]+\\.[0-9]+\\.[0-9]+'"]
+   ```
+   
+   In configuring commands, keep the following in mind:
+   
+   - You must specify the path to a valid executable program or file. 
```

Comment:
This is not true. `command: [hostname]` works just fine without specifying a path.

I think we mean to say that the executable must be located in `$PATH` or be an absolute path.